docs: Search and Recover Experiments of Chaosd

en-jin19 opened the pull request:
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

Translated from:
- https://github.com/chaos-mesh/website/pull/161
- https://github.com/chaos-mesh/website/pull/174

en-jin19 commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
 ---
-title: Search and Recover Chaosd's Experiment
+title: Search and Recover Experiments Using Chaosd
```
@WangXiangUSTC Could you please help check whether the title is correct? If so, I will revise the corresponding Chinese title, thansks!

ran-huang commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
+
+### Search experiments using service mode
+
+Currently, the service mode only supports searching all experiments. You can get the data by visiting the /api/experiments/ path of Chaosd service.
```
```suggestion
Currently, the service mode only supports searching all experiments. You can get the data by visiting the `/api/experiments/` path of Chaosd service.
```

ran-huang commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
+
+You can recover an experiment by using Chaosd recover UID.
+
+The following is an example of recovering an experiment using above way under the command-line mode.
```
```suggestion
The following example shows how to recover an experiment using the command-line mode.
```

ran-huang commented on docs/chaosd-search-recover.md:
````diff
@@ -1,5 +1,137 @@
+Attack stress cpu successfully, uid: 4f33b2d4-aee6-43ca-9c43-0f12867e5c9c
+```
+
+You need to pay attention to save the experiment UID in the output.
````
```suggestion
Save the experiment UID for later use.
```

ran-huang commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
+
+### Recover experiments using service mode
+
+You can recover an experiment by sending a `DELETE HTTP` request to the /api/attack/{uid} path of Chaosd service.
```
```suggestion
You can recover an experiment by sending a `DELETE` HTTP request to the `/api/attack/{uid}` path of Chaosd service.
```

ran-huang commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
+
+You can recover an experiment by sending a `DELETE HTTP` request to the /api/attack/{uid} path of Chaosd service.
+
+The following is an example of recovering an experiment using the above way under the service mode.
```
```suggestion
The following example shows how to recover an experiment using the service mode.
```

ran-huang commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,137 @@
+
+The following is an example of recovering an experiment using the above way under the service mode.
+
+1. Send an `HTTP POST` request to the Chaosd service to create a CPU stress experiment:
```
```suggestion
1. Send a `POST` HTTP request to the Chaosd service to create a CPU stress experiment:
```

ran-huang commented on docs/chaosd-search-recover.md:
````diff
@@ -1,5 +1,137 @@
+{"status":200,"message":"attack successfully","uid":"c3c519bf-819a-4a7b-97fb-e3d0814481fa"}
+```
+
+You need to pay attention to save the experiment UID in the output.
````
```suggestion
Save the experiment UID for later use.
```

ran-huang commented on i18n/zh/docusaurus-plugin-content-docs/current/chaosd-search-recover.md:
```diff
@@ -3,15 +3,15 @@ title: 查找和恢复 Chaosd 实验
 ### 使用命令行模式查找实验
 
-运行以下命令可查看搜索实验支持的配置：
+运行以下命令可查看当前搜索的实验所支持的配置：
```
```suggestion
运行以下命令可查看搜索命令 (`search`) 所支持的配置：
```

en-jin19 commented on i18n/zh/docusaurus-plugin-content-docs/current/chaosd-search-recover.md:
```diff
@@ -3,18 +3,18 @@ title: 查找和恢复 Chaosd 实验
 summary: 本文档介绍了查找和恢复 Chaosd 实验的操作方法，并提供了示例。
```
@WangXiangUSTC 如果英文中的 ”Search and Recover Experiments Using Chaosd“ 表达无误的话，中文标题以及文档对应内容会相应地改为：使用 Chaosd 查找和恢复实验

WangXiangUSTC replied:
我感觉 `使用 Chaosd 查找和恢复实验` 这个标题不太好，似乎暗示也可以用其他工具查找和恢复实验。实际上实验属于 chaosd，所以只能通过Chaosd 来查找和恢复。原来的标题的意思是实验属于 chaosd，如何对 chaosd 的实验进行查找和恢复

en-jin19 replied:
好的，谢谢，之前以为 Chaosd 是查找和恢复的工具，而不是实验是属于 Chaosd. 那么我统一在英语中修改为 experiments of Chaosd 吧 这样应该会更明确些

WangXiangUSTC replied:
好的

WangXiangUSTC commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,135 @@
+| :-- | :-- | :-- | :-- |
+| `all` | A | Lists all experiments | bool |
+| `asc` | None | Sorts the experiments in ascending order of the creation time. The default value is `false`.| bool |
+| `kind` | k | Lists experiments of the specified kind | string. The supported kinds are as follows: `network`, `process`, `stress`, `disk`, `host`, `JVM` |
```
```suggestion
| `kind` | k | Lists experiments of the specified kind | string. The supported kinds are as follows: `network`, `process`, `stress`, `disk`, `host`, `jvm` |
```

WangXiangUSTC commented on docs/chaosd-search-recover.md:
```diff
@@ -1,5 +1,135 @@
+| `all` | A | Lists all experiments | bool |
+| `asc` | None | Sorts the experiments in ascending order of the creation time. The default value is `false`.| bool |
+| `kind` | k | Lists experiments of the specified kind | string. The supported kinds are as follows: `network`, `process`, `stress`, `disk`, `host`, `JVM` |
+| `limit` | l | Lists the number of experiments | int |
```
should be `the number` of listed experiments`?

WangXiangUSTC commented on docs/chaosd-search-recover.md:
````diff
@@ -1,5 +1,135 @@
+./chaosd search --kind network --status destroyed --limit 1
+```
+
+By running this command, you can search the experiments of the kind of `network` in the status of `destroyed` (indicating that the experiment has been restored).
````
update `restored` to `recovered`

WangXiangUSTC commented on i18n/zh/docusaurus-plugin-content-docs/current/chaosd-search-recover.md:
```diff
@@ -35,22 +35,26 @@ Global Flags:
-| status ｜ s | 列出指定状态的实验 | string 类型，支持的状态包括：created、success、error、destroyed、revoked |
+| `all` | A | 列出所有的实验 | bool 类型 |
+| `asc` | 无 | 按照创建时间的升序对实验进行排列，默认值为 `false` | bool 类型 |
+| `kind` | k | 列出指定类型的实验 | string 类型，支持的类型包括：`network`、`process`、`stress`、`disk`、`host`、`JVM` |
```
JVM 需要小写

en-jin19 commented:
@WangXiangUSTC  I have accepted comments and updated the titles, PTAL, thanks!

@ran-huang I have updated the titles according to the tech review, PTAL, thanks!

WangXiangUSTC commented on docs/chaosd-search-recover.md:
```diff
@@ -39,8 +39,8 @@ Global Flags:
-| `kind` | k | Lists experiments of the specified kind | string. The supported kinds are as follows: `network`, `process`, `stress`, `disk`, `host`, `JVM` |
-| `limit` | l | Lists the number of experiments | int |
+| `kind` | k | Lists experiments of the specified kind | string. The supported kinds are as follows: `network`, `process`, `stress`, `disk`, `host`, `jvm` |
+| `limit` | l | The number of listed experiments? | int |
```
seems that the `?` can be deleted

WangXiangUSTC reviewed (commented):
rest LGTM

en-jin19 commented:
@WangXiangUSTC all comments are fixed, PTAL, thanks!

WangXiangUSTC reviewed (approved):
LGTM

WangXiangUSTC commented:
/merge
